Scripts/Debug/Yotam LCA Model.py

```diff
@@ -11,12 +11,10 @@
 import psyneulink as pnl
 import networkx as nx
 import sys
-import warnings
-warnings.filterwarnings("error", category=UserWarning)
```

Comment:
I think this should fix the 'message' regexp filter to only match the 10000 overflow warning rather than removing it entirely.